src/main/java/net/minecraftforge/client/event/EntityRenderersEvent.java

```diff
@@ -144,4 +149,36 @@ public EntityModelSet getEntityModels()
             return entityModels;
         }
     }
+
+    /**
+     * Provides a safe place to register models for the skull block
+     */
+    public static class CreateSkullModels extends EntityRenderersEvent
+    {
+        private final Builder<Type,SkullModelBase> builder;
+        private final EntityModelSet entityModelSet;
+
+        public CreateSkullModels(Builder<Type,SkullModelBase> builder, EntityModelSet entityModelSet)
+        {
+            this.builder = builder;
+            this.entityModelSet = entityModelSet;
+        }
+
+        public EntityModelSet getEntityModelSet()
+        {
+            return  entityModelSet;
```

Comment:
Formatting: duplicate space.
```suggestion
            return entityModelSet;
```